File: cmstestsuite/tasks/batch_fileio_managed/code/grader.cpp
#include <stdio.h>

extern int userfunc(int x);

int main() {
    int n;
    FILE *in = fopen("input.txt", "r");
    FILE *out = fopen("output.txt", "w");
    fscanf(in, "%d", &n);
    fprintf(out, "correct %d\n", userfunc(n));
    return 0;
}

File: cmstestsuite/tasks/batch_fileio_managed/code/grader.pas
program grader;
uses batchfileiomanaged;

var
    n: longint;
    infile, outfile: text;

begin
    assign(infile, 'input.txt');
    reset(infile);
    assign(outfile, 'output.txt');
    rewrite(outfile);

    readln(infile, n);
    write(outfile, 'correct ');
    writeln(outfile, userfunc(n));

    close(infile);
    close(outfile);
end.
